resources/docs/livewire/creating-chirps.md

```diff
@@ -0,0 +1,504 @@
+
+use Illuminate\Http\Request; // [tl! remove]
+use Illuminate\Http\Response; // [tl! add]
+
+class ChirpController extends Controller
+{
+    /**
+     * Display a listing of the resource. [tl! add:-1,7]
+     */
+    public function index(): Response
+    {
+        return response('Hello, World!');
+    }
+}
+```
+
+If you are still logged in from earlier, you should see your message when navigating to [http://localhost:8000/chirps](http://localhost:8000/chirps), or [http://localhost/chirps](http://localhost/chirps) if you're using Sail!
+
+## Livewire
+
+Not impressed yet? Let's update the `index` method of our `ChirpController` class to render a Blade view:
+
+```php filename=app/Http/Controllers/ChirpController.php
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Response;// [tl! remove]
+use Illuminate\View\View;// [tl! add]
+
+class ChirpController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(): Response// [tl! remove]
+    public function index(): View// [tl! add]
+    {
+        return response('Hello, World!');// [tl! remove]
+        return view('chirps', [// [tl! add:start]
+            //
+        ]);// [tl! add:end]
+    }
+}
+```
+
+We can then create our Blade view template with a Livewire component with a form for creating new Chirps:
+
+```blade filename=resources/views/chirps.blade.php
+<x-app-layout>
+    <div class="max-w-2xl mx-auto p-4 sm:p-6 lg:p-8">
+        <livewire:chirps.create />
+    </div>
+</x-app-layout>
+```
+
+And finally, we can create a Livewire component to render the form. For that, you may use the `make:volt` Artisan command.
+
+Note that the snippet below offers two different ways to create the component: one using a `Class` API and the other using a `Functional` API. You will see both APIs throughout this tutorial, and you can choose the one you prefer:
+
+```shell tab=Class
+php artisan make:volt chirps/create --class
+```
+
+```shell tab=Functional
+php artisan make:volt chirps/create
+```
+
+This will create a new Livewire component at the `app/resources/views/livewire/chirps/create.blade.php` path.
+
+Let's update the Livewire component contents to display a form:
+
+```php tab=Class filename=resources/views/livewire/chirps/create.blade.php
+<?php
+
+use Livewire\Volt\Component;
+
+new class extends Component
+{
+    public string $message = ''; // [tl! add]
+} ?>
+
+<div>
+    <form wire:submit="store"> <!-- [tl! add:start] -->
+        <textarea
+            wire:model="message"
+            placeholder="{{ __('What\'s on your mind?') }}"
+            class="block w-full border-gray-300 focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50 rounded-md shadow-sm"
+        ></textarea>
+
+        <x-input-error :messages="$errors->get('message')" class="mt-2" />
+        <x-primary-button class="mt-4">{{ __('Chirp') }}</x-primary-button>
+    </form> <!-- [tl! add:end] -->
+</div>
```

Comment:
Not a big deal, but the scaffolded file includes a `//` that isn't shown here. Beginners may not know what to do with it.

Unrelated to this PR, the `//` isn't a valid HTML comment so it actually renders in the browser. I wonder if the stubs should be updated to use `<!-- -->` or just remove it entirely.

Reply:
Fixed. Going to keep your `<!-- -->` feedback in main for volt.